content/en/docs/concepts/security/index.md

```diff
@@ -12,565 +12,543 @@ aliases:
     - /docs/concepts/policies/
 ---
 
-Breaking down a monolithic application into atomic services offers various benefits, including better agility, better scalability
-and better ability to reuse services.
-However, microservices also have particular security needs:
-
-- To defend against the man-in-the-middle attack, they need traffic encryption.
-
-- To provide flexible service access control, they need mutual TLS and fine-grained access policies.
-
-- To audit who did what at what time, they need auditing tools.
-
-Istio Security tries to provide a comprehensive security solution to solve all these issues.
-
-This page gives an overview on how you can use Istio security features to secure your services, wherever you run them.
-In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
-
-{{< image width="80%" link="./overview.svg" caption="Istio Security Overview" >}}
-
-The Istio security features provide strong identity, powerful policy, transparent TLS encryption, and authentication, authorization
-and audit (AAA) tools to protect your services and data. The goals of Istio security are:
-
-- **Security by default**: no changes needed for application code and infrastructure
+Breaking down a monolithic application into atomic services offers various
+benefits, including better agility, better scalability and better ability to
+reuse services. However, microservices also have particular security needs:
+
+- To defend against man-in-the-middle attacks, they need traffic encryption.
+- To provide flexible service access control, they need mutual TLS and
```

Comment:
Should mutual TLS be in the glossary, or maybe a link here (first use)?

Reply:
@frankbu We should add mutual TLS to the glossary as a separate PR and add the shortcode to all pages that need it.